digitransit-component/packages/digitransit-component-suggestion-item/src/index.js

```diff
@@ -8,13 +8,16 @@ import styles from './helpers/styles.scss';
 
 const BUS_EXPRESS = 702;
 const BUS_LOCAL = 704;
+const SPEEDTRAM = 902;
```

Comment:
Maybe we should call this TRAM_LOCAL in code instead of SPEEDTRAM as the spelling is more inline with the BUS_LOCAL and the 902 route type refers to Local Tram Service (even though in our context it means "pikaratikka").